torch/optim/_functional.py

```diff
@@ -357,3 +357,48 @@ def asgd(params: List[Tensor],
             ax.add_(param.sub(ax).mul(mu))
         else:
             ax.copy_(param)
+
+
+def nadam(params: List[Tensor],
```

Comment:
does this belong to this PR? looks like #59009 got added here?

Reply:
it belongs to #59009   :